packages/circuit-ui/components/Checkbox/Checkbox.tsx

```diff
@@ -241,29 +235,33 @@ export const Checkbox = forwardRef(
     const handleChange = useClickEvent(onChange, tracking, 'checkbox');
 
     return (
-      <CheckboxWrapper className={className} style={style} noMargin={noMargin}>
-        {/* @ts-expect-error the noMargin prop is required */}
-        <CheckboxInput
-          {...props}
-          id={id}
-          name={name}
-          value={value}
-          type="checkbox"
-          disabled={disabled}
-          invalid={invalid}
-          ref={ref}
-          onChange={handleChange}
-        />
-        <CheckboxLabel htmlFor={id} disabled={disabled}>
-          {children}
-          <Checkmark aria-hidden="true" />
-        </CheckboxLabel>
+      <>
         {!disabled && validationHint && (
-          <CheckboxTooltip position={'top'} align={'right'}>
-            {validationHint}
-          </CheckboxTooltip>
+          <Tooltip text={validationHint} placement="top-end" />
         )}
```

Comment:
I wonder if the Tooltip component is still suitable for this kind of use case where its visibility is not (directly) controlled by the user 🤔